docs/ReleaseNotes.md

```diff
@@ -1,289 +1,401 @@
+---
+id: ReleaseNotes
+title: Release Notes
+---
+
+## 2020.11.0 - 11/24/2020
+
+### New Features
+
+#### CAD and Video Asset Support
+
+##### Uploading CAD Files
+Do you use Enklu Cloud for your rapid prototyping needs? Are you interested in editing your CAD files in the web editor, without the extra hassle of exporting them from Unity?
+
+Starting with 2020.11, Enklu Cloud now supports direct imports for the following CAD formats:
+- .asm 
+- .sldasm, .sldasm.1 (in a .zip file with any dependencies)
+- .prt, .prt.1
+- .sldprt, .sldprt.1
+- .dwg
+- .dxf
+- .fbx
+- .obj
+- .pxz (in a .zip file with any dependencies)
+- .gltf
+- .3dm
+- .stl 
+
+And this is only the beginning! We'll be continuing to update Enklu Cloud to accommodate more and more CAD formats as time goes on. So if you use a format that isn't represented above, please [contact us](/contact)!
+
+![CAD Assets](/img/product/releases/2020.11/CAD_asset_support.gif)
+
+
+##### Better Video Support
+
+Aside from CAD support, we've also added better video support in 2020.11. You can now upload the following video format directly to Enklu Cloud:
+
+ - .asf
+ - .avi
+ - .dv
+ - .m4v
+ - .mov
+ - .mp4
+ - .mpg
+ - .mpeg
+ - .ogv
+ - .vp8
+ - .webm
+ - .wmv 
+
+You can also directly embed video URLs into your projects [using VineML](API/Vines.md#video).
+
+> **Note:** Videos currently will not play in the web editor, but they will play as expected on supported devices.
+
+
+![Better Video Support](/img/product/releases/2020.11/video_asset_support.gif)
+
+
+
+### Improvements
+
+#### My Experiences Screen
+
+In this update, we've revamped the entire My Experiences screen, making it faster than ever to find and create experiences! Not only can you create new projects and modify existing ones, but you can also search for them directly.
+
+We've also added user portraits on each experience card to tell you who has access to a file (you can check their names on mouseover). 
+
+Finally, we've also added the ability to sort your experiences by either Name or Recent Activity. No more are the days of spending minutes chasing down a file you just started working on!
+
+![My Experience Screen](/img/product/releases/2020.11/my_experiences_screen.gif)
+
+
+#### Drag & Drop Scripts
+
+We've added a bunch of new Public Scripts in 2020.11 that makes building interactions easier than ever! These Drag & Drop scripts allow even the least technical of folks to create complex simulations or immersive content. 
+
+To use one of these scripts, create a new asset and drag one of these  over to the Script section of the Inspector. You can then adjust different parameters, like the icon or color of a button, in the Variable window. Most importantly, you can also decide what message an interactable takes, and what message it sends on completion.  You could create a button that pulls apart a motor, or a proximity trigger that reveals a hidden message! 

```

Comment:
is interactable real a word? I mean, I understand the meaning, but unless we capitalize it a make it a term, I'd suggest something like "interactive element." Also, I'm not sure most people will understand messages in this context unless they're already power users. Maybe use the word "event" here instead. Like, "you can also decide what event an interactive element reacts to and what event it broadcasts on completion"